configs/guided_diffusion/adm-u_8xb32_imagenet-512x512.py

```diff
@@ -0,0 +1,45 @@
+_base_ = [
```

Comment:
Can this config inherit from `adm-u_8xb32_imagenet-64x64.py`?

Reply:
In fact, there are many differences between two configs (model, diffuser, dataset). So this config won't inherit from ``adm-u_8xb32_imagenet-64x64.py``?